web-app/src/App.js

```diff
@@ -16,6 +17,7 @@ const App = () => (
       <Route exact path="/" component={Home} />
       <Route path="/home" component={Home} />
       <Route path="/sights" component={Search} />
+      <Route path="/sightForm" component={SightForm} />
```

Comment:
That should probably be more like `add-sight` or something in that direction (add/sight or sights/add) but not Form and not camel case.